.editorconfig

```diff
@@ -10,3 +10,6 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[package*.json]
+indent_style = unset
```

Comment:
Prevent EditorConfig from messing with the `package.json` and `package-lock.json`